### PR TITLE
added OPBEANS_SERVICES environment variable

### PR DESCRIPTION
**Pull request by beniwohli**
refs #196

### CODE REVIEW

File: scripts/compose.py [hunk 1]
```diff
@@ -1758,6 +1758,13 @@ def start_handler(self):
         services = {}
         for service in selections:
             services.update(service.render())
+
+        # expose a list of enabled opbeans services to all opbeans services. This allows them to talk amongst each other
+        # and have a jolly good distributed time
+        enabled_opbeans_services = [k for k in services.keys() if k.startswith("opbeans-") and not k in ("opbeans-rum", "opbeans-load-generator")]
+        for s in enabled_opbeans_services:
+            services[s]["environment"].append("OPBEANS_SERVICES=" + ",".join(enabled_opbeans_services))
```
Comment (graphaelli) [inline scripts/compose.py]: environment can also be a `dict` - not sure if any opbeans services have that, yet

Comment (beniwohli) [reply scripts/compose.py]: fixed in 4e4f3c5

File: scripts/compose.py [hunk 2]
```diff
@@ -1758,6 +1758,17 @@ def start_handler(self):
         services = {}
         for service in selections:
             services.update(service.render())
+
+        # expose a list of enabled opbeans services to all opbeans services. This allows them to talk amongst each other
+        # and have a jolly good distributed time
+        enabled_opbeans_services = [k for k in services.keys() if k.startswith("opbeans-") and not k in ("opbeans-rum", "opbeans-load-generator")]
```
Comment (kuisathaverat) [inline scripts/compose.py]: maybe we want only to enable DT on some test, we could add a parameter for that something like this

if options.get('opbeans_dt_enable'):

Comment (beniwohli) [reply scripts/compose.py]: how about we kill two birds with one stone* and use a `--opbeans-dt-probability` float flag? Setting it to 0 disables dt, default will be 0.5

* no birds have been killed during the production of this comment

Comment (beniwohli) [reply scripts/compose.py]: @kuisathaverat implemented in 4d1df35

File: scripts/compose.py [hunk 3]
```diff
@@ -1758,6 +1772,18 @@ def start_handler(self):
         services = {}
         for service in selections:
             services.update(service.render())
+
+        # expose a list of enabled opbeans services to all opbeans services. This allows them to talk amongst each other
+        # and have a jolly good distributed time
+        enabled_opbeans_services = [k for k in services.keys()
+                                    if k.startswith("opbeans-") and k not in ("opbeans-rum", "opbeans-load-generator")]
```
Comment (alvarolobato) [inline scripts/compose.py]: @beniwohli does this include the ports? I'm testing with opbeans java and I'm not getting the ports.

**Comment by beniwohli** [comment]
Test failure  is unrelated, merging

**Pull request merged**